Apps/Sandcastle/gallery/Camera.html

```diff
@@ -108,14 +108,15 @@
 
     // View in east-north-up frame
     var camera = viewer.camera;
-    Cesium.Matrix4.clone(transform, camera.transform);
     camera.constrainedAxis = Cesium.Cartesian3.UNIT_Z;
 
     // Zoom in
-    camera.lookAt(
-        new Cesium.Cartesian3(-120000.0, -120000.0, 120000.0),
-        Cesium.Cartesian3.ZERO,
-        Cesium.Cartesian3.UNIT_Z);
+    camera.lookAt({
+        target : center,
+        offset : new Cesium.Cartesian3(-120000.0, -120000.0, 120000.0),
+        transform : transform
+    });
+    camera.setTransform(transform);
```

Comment:
It's not clear to me why we need to call `setTransform` here but not in the `3D Models.html` above.  Perhaps one is wrong?  In general, will users be able to figure out what it means to pass a `transform` to `lookAt` vs. call `setTransform`?